lxd/instance/drivers/driver_utils.go

```diff
@@ -0,0 +1,46 @@
+package drivers
+
+import (
+	"sync"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/lxc/lxd/shared"
+)
+
+// have we setup shared mounts?
+var sharedMounted bool
+var sharedMountsLock sync.Mutex
+
+// SetupSharedMounts sets up shared mounts.
+func SetupSharedMounts() error {
```

Comment:
I think it'd be cleaner to have `daemon.go` do this in a non-fatal way (log a warning).

Then have a check in container_lxc and refuse startup if not present with the current error.

Reply:
Will do